Cohort tech setup & Leading cohort call updates (#45)
Openscapes/approach-guide: champions/pre-cohort-tech.qmd

In this page, we give some general information, then describe things in temporal order.

### Project management

We use the [CohortPlanning GitHub Project](https://github.com/orgs/Openscapes/projects/7) to manage the tasks involved in running Openscapes Champions Cohorts (since June 2022) and have created some [Issue Templates](https://github.com/Openscapes/cohort-planning/tree/main/.github/ISSUE_TEMPLATE) with checklists. These can be used to quickly set up Issues for new Cohorts.

### `kyber` helps automate some setup

> If you're comfortable following instructions in R (or want to try!), we recommend using our [`kyber` R package](https://github.com/openscapes/kyber) created by [Sean Kross](https://seankross.com/). `kyber` contains tools for setting up learning cohorts on GitHub, purpose-built for the Openscapes Champions Program. It automates creation of new GitHub repos and agendas in Google Docs for example, using information from the Cohort Registry (described below). We began using `kyber` in June 2022, starting with the 2022-sefsc-summer Cohort.

Reviewer: ```suggestion
If you're comfortable following instructions in R (or want to try!), we recommend using our [`kyber` R package](https://github.com/openscapes/kyber) created by [Sean Kross](https://seankross.com/). `kyber` contains tools for setting up learning cohorts on GitHub, purpose-built for the Openscapes Champions Program. It automates creation of new GitHub repos and agendas in Google Docs for example, using information from the Cohort Registry (described below). We began using `kyber` in January 2022, starting with the 2022-afsc winter Cohort.
```

Specific `kyber` functions will be described in more detail where relevant to specific tasks below.

### Google Workspace

In June 2022 we began moving our work to an Openscapes Google Workspace, starting with the 2022 NOAA SEFSC summer Cohort.

**Folder organization**

-   OpenscapesMain \> OpenscapesChampionsProgram \> contains materials for active and completed Cohorts, including 1-on-1's and specific Cohort Folders.
-   OpenscapesMain \> OpenscapesCohortPlanning \> contains materials for planning Cohorts, including the Cohort Registry and docs for Mentors Seaside Chats.

**Calendar**: The OpenscapesCohorts Google Calendar is used for events, like Cohort Calls and Coworking sessions. Add CohortFolder to calendar invites; we learned that folks go to Calendars to find the docs for meetings.

There are limits to the number of Guests that can be added to a calendar invite. Google does not provide information on the actual limit number. Our recommended strategy is to add \~80 Guests initially, then add additional blocks of 40 Guests on subsequent days. We learned this for Cross-Cohort Coworking sessions for 4 NOAA NMFS Cohorts in Fall 2022.

::: {.callout-tip collapse="true"}

> ## Tip 

Author: ```suggestion
## Tip: Click URL to add to your Google Calendar 
```

> Sometimes we ask people add an event to their own Google Calendar without being added explicitly as a Guest. To do this, in a Calendar Event > click the 3 dots > Publish event > choose Copy Link to event and share link in Slack or email saying "Add this to your Google Calendar". We used this to let people in a Cohort know of an optional impromptu session like drop-in GitHub help, or a screenshare-and-tell for example.

Author: ```suggestion
Sometimes we ask people add an event to their own Google Calendar without being added explicitly as a Guest. To do this, in a Calendar Event > click the 3 dots > Publish event > choose Copy Link to event and share link in Slack or email saying "Click URL to add this to your Google Calendar". We have used this to let people in a Cohort know of an optional impromptu session like drop-in GitHub help, or a screenshare-and-tell for example.
```
